### DISCUSSION TITLE
Blocked by an object

### DISCUSSION THREAD

**Discussion by JBReaper95** [Courseplay/Courseplay_FS22#2765] [Q&A]
I was wondering why my harvester is saying that it is being blocked by an object when there is nothing in front of the harvester. I have not changed any of my settings and I have cut this field before with the same set up. Any ideas on what I have done or what I can do to solve this issue?
![Screenshot 2023-09-12 194048](https://github.com/Courseplay/Courseplay_FS22/assets/144858480/14316cec-6a45-4fd7-9c82-118e3d8ab87b)

**Comment by BalticHawk** [comment]
You could activate debug settings and post the log, so that the developers have more information.

**Reply by JBReaper95** [reply]
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/12592872/log.txt)
I have attached the log.txt of when the issue is happening. Thank you for the help

**Reply by Tensuko** [reply]
I guess this is a convert from FS19 with old colission wich will be detectd (can see that with F5 active ingame, orange box in arrow shape).
If that is the case, its a mod issue.

**Reply by JBReaper95** [reply]
I will have to check this when I get home, but what I find is weird is that I have used this setup for several harvests. I haven't changed any mods that I think would affect this. The only thing that really has changed is that I added straw to be created when harvesting corn. If it is a collision issue then all I have to do is adjust the collision box on the header? 

Thank you for your help with this issue.

**Reply by Tensuko** [reply]
What mod is this then? I mean the harvester.

**Reply by JBReaper95** [reply]
The harvester is the John Deere9600 - 9610 by SIID Modding and the header is the John Deere Corn Headers by SIID Modding. They are both on the mod hub. I have used this set up for 3 other seasons with no issues and I haven't changed anything. I did try using the in game header which is the same size and I have no issues so far. I haven't updated either mod so I am not sure what is causing this issue. Also thank you so much for your help!
